extensions/zeabur-docs/src/search-documentation.tsx

```diff
@@ -0,0 +1,60 @@
+import { List, ActionPanel, Action } from "@raycast/api";
+import { Language, Documentation } from "./type";
+import { getLanguages, getDocumentation } from "./util";
+import { useEffect, useState } from "react";
+
+export default function Command() {
+  const [languages, setLanguages] = useState<Language[]>([]);
+  const [documentation, setDocumentation] = useState<Documentation>({});
+  const [currentLanguage, setCurrentLanguage] = useState<string>("");
+
+  useEffect(() => {
```

Comment:
Could maybe be [useCachedPromise](https://developers.raycast.com/utilities/react-hooks/usecachedpromise) instead which also gives you isLoading